lingshare/lingshare-dev
# Wordlist new UI no auto save, create and edit completely done.

## NSF-1345 - https://speakagent.atlassian.net/browse/NSF-1345
### As an Author I want to be able to copy a wordlist that I do not own so that I can customize and assign it to my students

**Test Steps:**
1. Go to Wordlist dashboard
2. Click on the Community Tab
3. Press Copy and make sure a copy have been created
![GitHub Logo](http://g.recordit.co/dck4RgvY9c.gif)

## NSF-1357 - https://speakagent.atlassian.net/browse/NSF-1357
### Create tabbed interface css
1. Go to Wordlist dashboard
2. Make sure the tabs work
3. When the user search for a wordlist, the title of the tab should show the count of the result.
![GitHub Logo](http://g.recordit.co/4ce8YvuS3Z.gif)

## NSF-1316 - https://speakagent.atlassian.net/browse/NSF-1316
### As an Author I want the Add Words process integrated into the Wordlist process so that it's easy to use.

#### Add Words
1. Go to Wordlist dashboard
2. Click on "Edit" on any wordlist
3. Click on "Add Words"
4. Search the word and click in the result
5. When the user clicks on "Done" the words will be added to the list.
![GitHub Logo](http://recordit.co/bWrmrqcOtm.gif)


## NSF-1319 - https://speakagent.atlassian.net/browse/NSF-1319
### [UI] As an Author I want a helpful user interface to create, read, update and delete individual words and wordlists using the new UI.

#### Create a Wordlist
1. Go to Wordlist dashboard
2. Pres "+ Add Wordlist"
3. Fill the form and press "Save Wordlist" 
4. Make sure everything was inserted.
![GitHub Logo](http://g.recordit.co/MLmRwENbzC.gif )

#### Filter and Search Wordlist
1. Go to Wordlist dashboard
2. Go to "Your Wordlist" Tab
3. User the filters.
4. Search for a wordlist (this works with the filters  too)
![GitHub Logo](http://g.recordit.co/QVXQoQ74R6.gif)

#### Delete Words
1. Go to Wordlist dashboard
2. Click on "Edit" on any wordlist
3. On any words click on delete ( it should mark the image and line through the title).
4. Search for a wordlist (this works with the filters too), you can restore the word just in case it doesn't want to be deleted.
5. Click on "Save Edits"
![GitHub Logo](http://g.recordit.co/ilSJUGV8b3.gif)

#### Edit a Wordlist
1. Go to Wordlist dashboard
2. Click on "Edit" on any wordlist
3. Edit the wordlist Status, Grades, Subjects or Case
4. Click on "Save Edits"
![GitHub Logo](http://g.recordit.co/98T9Qv7T7o.gif)

#### Add or Delete Tags
1. Go to Wordlist dashboard
2. Click on "Edit" on any wordlist
3. Add tags by comma separated.
4. Delete any tag (deleted tags can be restored)
5. Click on "Save Edits"
![GitHub Logo](http://g.recordit.co/GpFyCq1FUA.gif)
